activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fixes assignment of frozen values in HashWithIndifferentAccess.
+    Fixes #18550
+
+    *Aditya Kapoor*
+
```

Comment:
Can you reword this a bit - something along the lines of:

``` markdown
Duplicate frozen array when assigning it to a HashWithIndifferentAccess so 
that it doesn't raise a `RuntimeError` when calling `map!` on it in `convert_value`.

Fixes #18550.

*Aditya Kapoor*
```

Once that's done it should be good to merge.